its/pom.xml

```diff
@@ -32,7 +32,7 @@
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
       <artifactId>sonar-orchestrator</artifactId>
-      <version>3.40.0.183</version>
+      <version>3.42.0.312</version>
```

Comment:
This update was needed. The old orchestrator version was adding the `sonar.login` property automatically.

Reply:
Educational: Since we still use the `sonar.login` property (SonarCloud compatibility reasons), you could technically leave it as it is, right? You just wanted to document why the tests use login instead of token?